src/pages/PointOfSale/BillTemplate.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+	<div class="bill">
```

Comment:
fix html formatting

Reply:
Fixed the HTML Formatting and fixed all style issues with ESLint